### DISCUSSION TITLE
Help Wanted! How to: Columns with delimiters -> Column without delimiter?

### DISCUSSION THREAD

**Discussion by a-y-u-s-h** [saulpw/visidata#1219] [Q&A] [answered]
How would you go about generating multiple rows based on some delimiter specified in a column like so?

![example](https://user-images.githubusercontent.com/19586719/143668736-4fc78cc5-2908-450e-9de8-470bdc416774.PNG)

This seems to be a common pattern in data in most places. So much so that tools like PowerBI have [a method](https://community.powerbi.com/t5/Desktop/duplicate-rows-when-column-has-double-value-separate-by-comma/td-p/1124408) for it as well. I can't seem to figure out how to do this in visidata.

**Comment by geekscrapy** [comment, marked as answer]
While you have `Column B` selected do:
`=curcol.split(",")`

Then you can use `unfurl-col` on that new column 👍

**Reply by a-y-u-s-h** [reply]
That's handy. Can I also turn this combination into a keybinding somehow? Also where do I find documentation about `curcol`, I have absolutely no idea how this works.

**Reply by saulpw** [reply]
`curcol` just means "the column the cursor is on now" (and it doesn't move with the future cursor).  You could also use the column name itself, it's just that `curcol` will work even if the column has a weird name, or if you want to use it in a macro or something.

**Reply by frosencrantz** [reply]
@a-y-u-s-h  be aware that `curcol` is only available in `addcol-expr`, the `=` command.  It doesn't work with other commands that allow the user to specify column names as part of expressions or commands.  I would like to see some interest from people to have this added.  If you also find that this would be useful, add a comment to the first [bug]( https://github.com/saulpw/visidata/issues/989).

- https://github.com/saulpw/visidata/issues/989
   - Also noticed by @geekscrapy  in https://github.com/saulpw/visidata/issues/1031
